### PR TITLE
Fix mainnet acp resync

### PR DESCRIPTION
**Pull request by ququzone**

### CODE REVIEW

File: packages/neuron-wallet/src/models/synced-block-number.ts [hunk 1]
```diff
@@ -87,7 +87,21 @@ export default class SyncedBlockNumber {
     if (!this.#liveCellBlockNumberEntity) {
       this.#liveCellBlockNumberEntity = new SyncInfoEntity()
       this.#liveCellBlockNumberEntity.name = SyncInfoEntity.CURRENT_LIVE_CELL_BLOCK_NUMBER
-      this.#liveCellBlockNumberEntity.value = assetAccountInfo.sudtDeployHeight.toString()
+      const sudtDeployHeight = assetAccountInfo.sudtDeployHeight.toString()
+      if (sudtDeployHeight != "-1") {
```
Comment (katat) [inline packages/neuron-wallet/src/models/synced-block-number.ts:91]: use strict `!==`

File: packages/neuron-wallet/src/models/asset-account-info.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ export default class AssetAccountInfo {
         codeHash: '0x6a3982f9d018be7e7228f9e0b765f28ceff6d36e634490856d2b186acf78e79b',
         hashType: ScriptHashType.Type
       }
-      this.sudtDeployHeight = BigInt(14922)
+      this.sudtDeployHeight = BigInt(-1)
```
Comment (katat) [inline packages/neuron-wallet/src/models/asset-account-info.ts:40]: Maybe setting this to null is more meaningful.

**Comment by katat** [comment]
@ququzone Please also reorganize the udt/acp settings to the configuration file `env`.

File: packages/neuron-wallet/package.json [hunk 3]
```diff
@@ -38,6 +38,7 @@
     "archiver": "3.1.1",
     "bn.js": "4.11.8",
     "chalk": "3.0.0",
+    "dotenv": "^8.2.0",
```
Comment (Keith-CY) [inline packages/neuron-wallet/package.json:41]:
```suggestion
    "dotenv": "8.2.0",
```

Comment (Keith-CY) [reply packages/neuron-wallet/package.json:41]: Isn't the `lock` file updated?

File: packages/neuron-wallet/.env [hunk 4]
```diff
@@ -0,0 +1,25 @@
+## mainnet
```
Comment (Keith-CY) [inline packages/neuron-wallet/.env:1]: Uppercases are preferable.

File: packages/neuron-wallet/.env [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+MAINNET_SUDT_DEP_TXHASH=0x0000000000000000000000000000000000000000000000000000000000000000
+MAINNET_SUDT_DEP_INDEX=0
+MAINNET_SUDT_DEP_TYPE=code
+MAINNET_SUDT_SCRIPT_CODEHASH=0x0000000000000000000000000000000000000000000000000000000000000000
+MAINNET_SUDT_SCRIPT_HASHTYPE=data
+MAINNET_SUDT_HEIGHT=-1
+MAINNET_ACP_DEP_TXHASH=0x0000000000000000000000000000000000000000000000000000000000000000
+MAINNET_ACP_DEP_INDEX=0
+MAINNET_ACP_DEP_TYPE=code
+MAINNET_ACP_SCRIPT_CODEHASH=0x0000000000000000000000000000000000000000000000000000000000000000
```
Comment (katat) [inline packages/neuron-wallet/.env:11]: Do we have to set these variables to a dummy values if they are not available yet on mainnet?
Can we just comment them out for now?

Comment (katat) [reply packages/neuron-wallet/.env:11]: I think it's better to leave them `NULL` value if these values are not available yet. Setting them to a dummy value like `0x0000000000000000000000000000000000000000000000000000000000000000 ` or `-1` could be misleading.

Comment (ququzone) [reply packages/neuron-wallet/.env:11]: `null` will cause type mismatch error

Comment (katat) [reply packages/neuron-wallet/.env:11]: Then we can change the type to be nullable.

Comment (katat) [reply packages/neuron-wallet/.env:11]: After a detailed look at the codes coupled with these variables, it does need some works to refactor these values to `nulls`. 

I will create an issue to remind the refactoring of these variables in a later stage.

**Comment by Keith-CY** [comment]
Please cherry-pick to the [rc/v0.31.0](https://github.com/nervosnetwork/neuron/tree/rc/v0.31.0)

**Pull request merged**